react-native-reanimated-tests.tsx

```diff
@@ -360,3 +363,33 @@ function WithDecayTest() {
     </PanGestureHandler>
   );
 }
+
+// useWorkletCallback
+function UseWorkletCallbackTest() {
+  const callback = useWorkletCallback((a: number, b: number) => {
+    return a + b;
+  });
```

Comment:
I should've to add deps here too.

```suggestion
  }, []);
```